controls/apache_spec.rb

```diff
@@ -92,22 +90,44 @@
     it { should_not be_writable.by('others') }
     it { should_not be_executable.by('others') }
   end
-  describe file(File.join(apache.conf_dir, '/conf-enabled/hardening.conf')) do
-    it { should be_owned_by 'root' }
-    it { should be_grouped_into 'root' }
-    it { should be_readable.by('owner') }
-    it { should be_writable.by('owner') }
-    it { should_not be_executable.by('owner') }
-    it { should be_readable.by('group') }
-    it { should_not be_writable.by('group') }
-    it { should_not be_executable.by('group') }
-    it { should_not be_readable.by('others') }
-    it { should_not be_writable.by('others') }
-    it { should_not be_executable.by('others') }
-  end
 end
 
 control 'apache-06' do
```

Comment:
If we check everything the the apache_conf resource, do we need these checks at all? The files themselves could be named/located anywhere.